### DISCUSSION TITLE
Verbose logs for CLI

### DISCUSSION THREAD

**Discussion by ch-sander** [oxigraph/oxigraph#1114] [Q&A]
Running oxigraph in a Docker container, starting the CLI like `oxigraph serve...` I was wondering if there is a flag to enable more verbose logging (currently basically `Listening for requests at http://0.0.0.0:7878` or `OxHTTP TCP error when writing response to 172.20.0.1:51972: Broken pipe (os error 32)`). I have queries leading to a 502 (`oxigraph-1 exited with code 0`, and restarts) for bad SPARQL (so, not oxigraph's mistake), but this error only occurs after hours (as the query takes that long, too). It would help for trouble shooting to get more insights.

**Comment by Tpt** [comment]
This is a great point! Sadly, there is no flag yet but indeed it would be great to have a verbose mode that does at least what you described.
